core/embed/sec/tropic/tropic.c

```diff
@@ -85,30 +86,22 @@ bool tropic_wait_for_ready(void) {
   return false;
 }
 
-lt_ret_t tropic_start_custom_session(const uint8_t *stpub,
-                                     const pkey_index_t pkey_index,
-                                     const uint8_t *shipriv,
-                                     const uint8_t *shipub) {
+void tropic_session_invalidate(void) {
+  g_tropic_driver.session_started = false;
```

Comment:
We should also call `lt_session_abort()`.

Reply:
Fixed in https://github.com/trezor/trezor-firmware/pull/6081/commits/0641821b7f9a79357b8f23dddc2e1bbe334ef898.